tests/Faithlife.Data.Tests/Faithlife.Data.Tests.csproj

```diff
@@ -3,6 +3,10 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;net472;net5.0</TargetFrameworks>
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
```

Comment:
`TargetFrameworks` is here twice.

Reply:
Merge error, probably a result of by `*.csproj text merge=union` in `.gitattributes`.